app/controllers/application_controller.rb

```diff
@@ -1,2 +1,5 @@
 class ApplicationController < ActionController::Base
+  def current_user
+    User.find(6)
+  end
```

Comment:
- [ ] Nice work implementing the `current_user` method :clap: However it would be nice to make it return the first user from the database (e.g. `User.first`) to follow the project requirement.
![Screenshot from 2023-12-07 16-26-49](https://github.com/Chimoneg27/Rails-Blog-App/assets/91142432/33987a6b-4512-4274-906a-c056d3af1899)